sdk/tests/deqp/framework/common/tcuTexLookupVerifier.js

```diff
@@ -741,7 +741,12 @@ goog.scope(function() {
         for (var i = minI; i <= maxI; i++) {
             /** @type {number} */ var x = tcuTexVerifierUtil.wrap(sampler.wrapS, i, level.getWidth());
             /** @type {number} */ var y = tcuTexVerifierUtil.wrap(sampler.wrapT, j, level.getHeight());
-            /** @type {Array<number>} */ var color = tcuTexLookupVerifier.lookupScalar(level, sampler, x, y, coordZ);
+            /** @type {Array<number>} */ var color;
+            if (tcuTexLookupVerifier.isSRGB(level.getFormat())) {
+                color = tcuTexLookupVerifier.lookupFloat(level, sampler, x, y, coordZ);
+            } else {
+                color = tcuTexLookupVerifier.lookupScalar(level, sampler, x, y, coordZ);
+            }
```

Comment:
If I'm understanding this fix correctly, don't all three of these functions need the same fix?

isNearestSampleResultValid_CoordXYAsNumber
isNearestSampleResultValid_CoordAsVec2AndInt (this change)
isNearestSampleResultValid_CoordAsVec3

?

I think the template code should have been ported differently, probably by making the template arguments real run-time arguments to the functions. However, for the moment, let's make the minimal fix as you've done, but let's do it everywhere needed. Thanks.